pkg/cmd/service/service_test.go

```diff
@@ -48,14 +43,10 @@ func TestMain(m *testing.M) {
 		panic(err)
 	}
 
-	config := DefaultConfig()
-	openFGAServerAddr = fmt.Sprintf("http://localhost%s", config.HTTP.Addr)
-	openFGAGRPCAddr = fmt.Sprintf("localhost%s", config.GRPC.Addr)
-
 	os.Exit(m.Run())
 }
 
-func ensureServiceUp(t *testing.T, transportCredentials credentials.TransportCredentials, httpHealthCheck bool) {
```

Comment:
I'd like to refactor this composition at some point. This is starting to get rough. I'm going to save that minor refactor for a later PR though. The main goal is to try and fix flaky tests for now.